js/util.js

```diff
@@ -169,8 +169,7 @@ function addWithCapEfficiency(a, cap, strength = 2) {
 
 function sumGeometricSeries(base, rate, n, owned = 0) {
     base *= rate ** owned;
-    if (n == 1) return base;
```

Comment:
This early return is still worth keeping because it prevents pointless exponentiation.